networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -132,6 +132,12 @@ void GraphGTest::SetUp() {
 
 /** CONSTRUCTORS **/
 
+TEST_F(GraphGTest, testDefaultConstructorIndexed) {
+    Graph G = Graph(3, false, false, true);
+    G.addEdge(0, 1);
+    ASSERT_EQ(0, G.edgeId(0, 1));
```

Comment:
In this case it is better to use `EXPECT_EQ` (see https://testing.googleblog.com/2008/07/tott-expect-vs-assert.html). Also, avoid yoda conditions (https://en.wikipedia.org/wiki/Yoda_conditions)
```suggestion
    EXPECT_EQ(G.edgeId(0, 1), 0);
```